### DISCUSSION TITLE
Chwalipięta

### DISCUSSION THREAD

**Discussion by hawkeye116477** [MajkiIT/polish-ads-filter#18305] [Show and tell]
Macie ładny pulpit swojego smartfona/peceta/konsoli albo może otrzymaliście najlepszy wynik w Saperze? Jeżeli tak, to jest to dobre miejsce by o tym opowiedzieć, gdyż tutaj chwalimy się wszystkim. 

------------------------
To może ja zacznę. Oto moja nowa konsola: 
<img src="https://cdn.mos.cms.futurecdn.net/vqQ3JKa7fC9GekYzmjB5TJ.jpg" width="60%" height="60%">

Wystarczy jedynie 5 minut rozgrywki w Cyberpunka 2077 w 4k/240 i mam gotowy obiad :smile:

**Comment by xxcriticxx** [comment]
KFC Xbox combo

**Comment by xxcriticxx** [comment]
![](https://5.imimg.com/data5/YN/GX/MY-28487095/philips-airfryer-250x250.jpg)
